lib/bridge/IrcBridge.js

```diff
@@ -560,6 +560,7 @@ IrcBridge.prototype.getThirdPartyProtocol = function(protocol) {
         },
         instances: servers.map((server) => {
             return {
+                network_id: server.domain,
```

Comment:
Can this change? e.g. if we change the domain we use to connect to a network? Its important that that this remains constant unless the AS republishes all the rooms after a change.